### PR TITLE
updated per minor tweak to docstring spec

### PR DESCRIPTION
**Pull request by cswiii**
Providing a minor update to the docstring spec that allows stuff to be better viewable in ReadTheDocs! :)

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -32,13 +32,19 @@ Expected Docstring format:
...
-     2.  Log in with valid user credentials
+    1.  Launch the url
+    2.  Log in with valid user credentials
+
     @Assert: Log in successful
+
     @BZ: #1234567
+
     @Status: Manual (REMOVE this field once automated)
     """
```
Comment (elyezer) [inline README.rst]: Blank line before this is required

File: README.rst [hunk 2]
```diff
@@ -32,13 +32,19 @@ Expected Docstring format:
 
     """
```
Comment (elyezer) [inline README.rst]: Move `@test` to here, just after `"""`

```python
"""@Test: Log in as a valid user

@Feature: Login

...

@Status: Manual

"""
```

**Comment by elyezer** [comment]
Also this needs a rebase

**Comment by sthirugn** [comment]
ACK pending @elyezer's comments

**Comment by Ichimonji10** [comment]
Please add a space between `@steps` and the beginning of the list. This:

    steps:
    1. one
    2. two

Yields this:

    <p>steps:
    1. one
    2. two</p>

This:

    steps:
    
    1. one
    2. two

Yields this:

    <p>steps:</p>
    <ol>
        <li>one</li>
        <li>two</li>
    </ol>

**Pull request closed without merging**